tests/conftest.py

```diff
@@ -869,6 +869,40 @@ def swapSyncd(request, duthosts, enum_rand_one_per_hwsku_frontend_hostname, cred
         if swapSyncd:
             docker.restore_default_syncd(duthost, new_creds)
 
+@pytest.fixture(scope='module')
+def swapQosSyncd(request, duthosts, enum_rand_one_per_hwsku_frontend_hostname, creds, tbinfo, lower_tor_host):
```

Comment:
Why do we need a new fixture? Can we not just add the check in L884 into the existing 'swapSyncd' fixture?

Reply:
There are another two methods refer to "swapSyncd()", as below:

```
@pytest.fixture(scope='module', autouse=True)
def prepare_syncdrpc(swapSyncd):
    pass

... omitted ...

class TestQosSaiMasic(QosSaiBaseMasic):

    def test_qos_masic_dscp_queue_mapping(
        self, duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_backend_asic_index,
        ptfhost, dut_test_params, swapSyncd, tbinfo
    ):
```

Before your comments, I worried about change's impact to "prepare_syncdrpc()" and "test_qos_masic_dscp_queue_mapping", so didn't touch original "swapSyncd()" to keep backward compatible.

Now, accept your suggestion and will run relevant test to verify it.